core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -270,63 +273,71 @@ fun LPGenericElevatedCard(
  */
 @Composable
 fun LPGenericElevatedCardImage(
-    imageUrl: String,
+    imageUrl: String?,
     title: String,
     description: String,
     onClick: () -> Unit
 ) {
+    val dp2 = LeasePertTheme.sizes.middleSize
+    val dp8 = LeasePertTheme.sizes.smallerSize
+    val dp12 = LeasePertTheme.sizes.midSmallSize
+    val dp175 = LeasePertTheme.sizes.extraSize175
```

Comment:
Another better way to call the sizes without create a variable for each one is doing something like this: `val sizes = LeasePertTheme.sizes` and then just use it like this: `sizes.middleSize`